From: Johannes Thumshirn <morbidrsa@googlemail.com>
Date: Wed, 27 Jun 2012 21:25:59 +0200
Subject: [035/106] staging: line6: changed interface of pod_send_channel()

commit 8d6b7f7c9b3c346f2fef496827c3fbbc2ebef1a9 upstream.

Adjusted interface of pod_send_channel() in order to take changes of
line6_send_program() into account.

Signed-off-by: Johannes Thumshirn <morbidrsa@googlemail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/line6/pod.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.2.46/drivers/staging/line6/pod.c
===================================================================
--- linux-3.2.46.orig/drivers/staging/line6/pod.c	2013-07-26 19:28:41.000000000 +0000
+++ linux-3.2.46/drivers/staging/line6/pod.c	2013-07-26 19:28:44.000000000 +0000
@@ -405,7 +405,7 @@
 /*
 	Send channel number (i.e., switch to a different sound).
 */
-static void pod_send_channel(struct usb_line6_pod *pod, int value)
+static void pod_send_channel(struct usb_line6_pod *pod, u8 value)
 {
 	line6_invalidate_current(&pod->dumpreq);
 
